### PR TITLE
Rds/kjw/function get predominant hvac building area type

### PR DESCRIPTION
**Pull request by KarenWGard**

### CODE REVIEW

File: docs/ruleset_functions/get_predominant_HVAC_building_area_type.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+**Returns:**  
+- **predominant_HVAC_building_area_type**: A String of the predominant HVAC building area type, defined as the building area type with the largest floor area
+ 
+**Function Call:** 
+- **get_HVAC_building_area_types_and_zones**
+
+## Logic:  
+
+- call get_HVAC_building_area_types_and_zones and sort the list: `sorted_list = sorted(get_HVAC_building_area_types_and_zones(RMR).items(), key=lambda x: x[1]["AREA"], reverse = True)`
+- the first item in the list will be the building area type with the largest area: `predominant_HVAC_building_area_type = sorted_list[0][0]`
```
[the remark this answers is not in the document]
Comment (KarenWGard) [reply docs/ruleset_functions/get_predominant_HVAC_building_area_type.md:19]: oh, good call.  What's the best way to issue a warning up the stack?  The math happens in a function that is called by other functions which are called by the rule test.

File: docs/ruleset_functions/get_predominant_HVAC_building_area_type.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# get_predominant_HVAC_building_area_type
+
+**Description:** references the `get_HVAC_building_area_types_and_zones` function, and then sorts the list by area
+- returns a String `predominant_HVAC_building_area_type` which is one of the HVAC building area types
+- used to verify the correct type of HVAC baseline system (or systems)
+
+**Inputs:**  
+- **RMR**
```
[the remark this answers is not in the document]
Comment (KarenWGard) [reply docs/ruleset_functions/get_predominant_HVAC_building_area_type.md:8]: I am using RDM / RMI now as discussed in the call

File: docs/ruleset_functions/get_predominant_HVAC_building_area_type.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (KarenWGard) [reply docs/ruleset_functions/get_predominant_HVAC_building_area_type.md:19]: get_HVAC_building_area_types_and_zones combines all building area types for the building

File: docs/ruleset_functions/get_predominant_HVAC_building_area_type.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (mkarpman) [reply docs/ruleset_functions/get_predominant_HVAC_building_area_type.md:8]: I still see RMR nomenclature being used. I will ignore this going forward and let you two figure it out :-).

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**